lib/k8s/config.rb

```diff
@@ -133,7 +133,7 @@ def self.from_kubeconfig_env(kubeconfig = nil)
     #
     # @param server [String] kubernetes server address
     # @param ca [String] server certificate authority data (base64 encoded)
-    # @param token [String] access token
+    # @param auth_token [String] access token
```

Comment:
Wrong attribute name, yardoc warns about these when generating, we could add rubocop-like test step for yardoc validity.